release-notes/opensearch-dashboards-maps.release-notes-2.8.0.0.md

```diff
@@ -0,0 +1,15 @@
+## Version 2.8.0.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.8.0
+
+### Features
+
+### Infrastructure
+
+### Refactoring
+
+### Enhancements
+
+### Bug Fixes
```

Comment:
Do we need this if we don't have any PR?

Reply:
probably we can remove, not sure about the format. Seems in other repos no PR - no section, so I'll do same here

Reply:
@VijayanB removed, please check